concourse/pipelines/windows-image-build-mbr.jsonnet

```diff
@@ -0,0 +1,346 @@
+};
+
+local ImgGroup(name, images, environments) = {
+  name: name,
+  jobs: [
+    'build-' + image
+    for image in images
+  ] + [
+    'publish-to-%s-%s' % [env, image]
+    for env in environments
+    for image in images
+  ],
+};
+
+// Start of output.
+{
+  local windows_2012_images = [
+    'windows-server-2012-r2-dc-mbr',
+    'windows-server-2012-r2-dc-core-mbr',
+  ],
+  local windows_2016_images = [
+    'windows-server-2016-dc-mbr',
+    'windows-server-2016-dc-core-mbr',
+  ],
+  local windows_2019_images = [
+    'windows-server-2019-dc-mbr',
+    'windows-server-2019-dc-core-mbr',
+  ],
+  local windows_2022_images = [
+    'windows-server-2022-dc-mbr',
+    'windows-server-2022-dc-core-mbr',
+  ],
+
+  local windows_server_images = windows_2012_images + windows_2016_images + windows_2019_images
+                              + windows_2022_images,
+
+  resource_types: [
+    {
+      name: 'gcs',
+      source: { repository: 'frodenas/gcs-resource' },
+      type: 'registry-image',
+    },
+    {
+      name: 'registry-image-forked',
+      type: 'registry-image',
+      source: { repository: 'gcr.io/compute-image-tools/registry-image-forked' },
+    },
+  ],
+  resources: [
+               common.GitResource('compute-image-tools'),
+               common.GitResource('guest-test-infra'),
+             ] +
+             [
+               common.GcsImgResource(image, 'windows-bios')
+               for image in windows_server_images
+             ] +
+             [
+               common.GcsSbomResource(image, 'windows-server')
+               for image in windows_server_images
+             ],
+  jobs: [
+          ImgBuildJob('windows-server-2022-dc-mbr', 'win2022-64', 'windows_gcs_updates_server2022'),
+          ImgBuildJob('windows-server-2022-dc-core-mbr', 'win2022-64', 'windows_gcs_updates_server2022'),
+          ImgBuildJob('windows-server-2019-dc-mbr', 'win2019-64', 'windows_gcs_updates_server2019'),
+          ImgBuildJob('windows-server-2019-dc-core-mbr', 'win2019-64', 'windows_gcs_updates_server2019'),
+          ImgBuildJob('windows-server-2016-dc-mbr', 'win2016-64', 'windows_gcs_updates_server2016'),
+          ImgBuildJob('windows-server-2016-dc-core-mbr', 'win2016-64', 'windows_gcs_updates_server2016'),
+          ImgBuildJob('windows-server-2012-r2-dc-mbr', 'win2012-r2-64', 'windows_gcs_updates_server2012r2'),
+          ImgBuildJob('windows-server-2012-r2-dc-core-mbr', 'win2012-r2-64', 'windows_gcs_updates_server2012r2'),
+        ] +
+
+        [
+          ImgPublishJob(image, env, 'windows', 'windows-bios')
+          for image in windows_server_images
+          for env in envs
+        ],
+
+  groups: [
+    ImgGroup('windows-2012-mbr', windows_2012_images, envs),
+    ImgGroup('windows-2016-mbr', windows_2016_images, envs),
+    ImgGroup('windows-2019-mbr', windows_2019_images, envs),
+    ImgGroup('windows-2022-mbr', windows_2022_images, envs),
```

Comment:
I don't believe we need to build 2022 as an MBR image- it never existed as MBR. WDYT?

Reply:
We need it because customers can import Windows Server 2022 MBR.

Reply:
Good point, or in place upgrades. Ok.